public/js/controllers/blocks.js

```diff
@@ -45,7 +50,8 @@ function displayBlocks() {
     const blockHtmlElements = blocks.slice(1).map(newBlockHtmlElement).join("\n");
     blocksHolder.append(blockHtmlElements);
 
-    figureNumberOfBlocksToDisplay();
+    // allow time for left panel to load in order to properly estimate available space on the right
+    setTimeout(figureNumberOfBlocksToDisplay, 500);
```

Comment:
This feels like an anti-pattern.  @gozart1 Is there a better way to handle this? Something with `async` perhaps?

Reply:
@gozart1 any thoughts here?